apps/block_scout_web/lib/block_scout_web/templates/layout/app.html.eex

```diff
@@ -30,8 +30,8 @@
     <%= render_existing(@view_module, "styles.html", assigns) %>
 
     <link rel="apple-touch-icon" sizes="180x180" href="<%= static_path(@conn, "/apple-touch-icon.png") %>">
-    <link rel="icon" type="image/png" sizes="32x32" href="<%= static_path(@conn, "/images/favicon-32x32.png") %>">
-    <link rel="icon" type="image/png" sizes="16x16" href="<%= static_path(@conn, "/images/favicon-16x16.png") %>">
+    <link rel="icon" type="image/png" sizes="32x32" href="<%= static_path(@conn, "/images/favicon-32x32.png?v=M44lzPylqQ") %>">
+    <link rel="icon" type="image/png" sizes="16x16" href="<%= static_path(@conn, "/images/favicon-16x16.png?v=M44lzPylqQ") %>">
     <link rel="manifest" href="<%= static_path(@conn, "/manifest.webmanifest") %>">
     <link rel="mask-icon" href="<%= static_path(@conn, "/safari-pinned-tab.svg") %>" color="#5bbad5">
```

Comment:
I think this will need to be updated for the favicon on Safari
`<link rel="mask-icon" href="<%= static_path(@conn, "/images/favicon-32x32.png") %>">`

Reply:
And if so, the `safari-pinned-tab.svg` file can be removed too

Reply:
Ok thanks for the insight, app file updated